testsuite/tests/typing-modules/nondep_private_abbrev.ml

```diff
@@ -149,5 +149,5 @@ module I : functor (X : sig end) -> sig type t = Priv(X).t end
 
 module IndirectPriv = I(struct end);;
 [%%expect{|
-module IndirectPriv : sig type t end
+module IndirectPriv : sig type t : value mod immutable with int end
```

Comment:
Yikes. `with int` is ugly. Definitely leave a CR for this one.

Reply:
Done.